README.md

```diff
@@ -218,6 +254,7 @@ The above command pulled the existing data out of Route53 and placed the results
 | [ZoneFileSource](/octodns/source/axfr.py) | | A, AAAA, CAA, CNAME, MX, NS, PTR, SPF, SRV, TXT | No | read-only |
 | [TinyDnsFileSource](/octodns/source/tinydns.py) | | A, CNAME, MX, NS, PTR | No | read-only |
 | [YamlProvider](/octodns/provider/yaml.py) | | All | Yes | config |
+>>>>>>> origin/master
```

Comment:
Looks like some merge conflict stuff hanging around here

Reply:
Doh, been so long since i resolved conflicts that I forgot how to do it :grin: